### DISCUSSION TITLE
Welcome to Discord Tickets Discussions!

### DISCUSSION THREAD

**Discussion by eartharoid** [discord-tickets/bot#77] [Announcements]
## 👋 Welcome!

Welcome to the Discord Tickets community!

You can create a discussion to:

- ✅ ask for help or other questions
- ✅ start a general conversation with the community
- ❌ report bugs - please [create an issue](https://github.com/discord-tickets/bot/issues/new/choose)
- ❌ request features - please [post on Feedbacky](https://lnk.earth/dsctickets-feedback)

> **Note**
> **If you're looking for help, [the Discord server](https://lnk.earth/discord) is much more active** than Discussions.

### How to use Discussions

When you start a discussion, please select the appropriate category and name your discussion topic with something descriptive.

**Comment by wh1337** [comment]
Hey @eartharoid! I was curious if you would be open to the idea of a PR to enabled docker for this repo? 😄

**Reply by eartharoid** [reply]
Sure
